packages/transport/src/transports/abstractUsb.ts

```diff
@@ -85,56 +85,55 @@ export abstract class AbstractUsbTransport extends AbstractTransport {
     }
 
     public acquire({ input }: { input: AcquireInput }) {
-        return this.scheduleAction(async () => {
-            const { path } = input;
+        return this.scheduleAction(
+            async () => {
+                const { path } = input;
+
+                if (this.listening) {
+                    this.listenPromise[path] = createDeferred<string>();
+                    this.listenPromise[path].promise
+                        .catch(err => err)
```

Comment:
Is this only an equivalent of empty catch? Because the value isn't returned anywhere, right?